apps/activemq.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apps
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/otel"
+)
+
+type MetricsReceiverActivemq struct {
+	confgenerator.ConfigComponent `yaml:",inline"`
+
+	confgenerator.MetricsReceiverShared `yaml:",inline"`
+
+	Endpoint                               string `yaml:"endpoint" validate:"omitempty,hostname_port|startswith=service:jmx:"`
+	Username                               string `yaml:"username" validate:"required_with=Password"`
+	Password                               string `yaml:"password" validate:"required_with=Username"`
+	confgenerator.MetricsReceiverSharedJVM `yaml:",inline"`
+
+	CollectJVMMetics *bool `yaml:"collect_jvm_metrics"`
```

Comment:
Can we use `MetricsReceiverSharedCollectJVM` here? The recent Elasticsearch PR added some common utils: https://github.com/GoogleCloudPlatform/ops-agent/pull/349/files